app/src/plugins/dates.ts

```diff
@@ -14,19 +14,28 @@
  * limitations under the License.
  */
 
+
 const DAY_MS = 24 * 60 * 60 * 1000;
 
-export function renderDaysAgo(lastUpdated: number) {
+export function renderMonthsAgo(lastUpdated: number) {
```

Comment:
If you change the name of this function you have to also change all of the places it was used.

Reply:
Oh yeah, right!